src/TableCell.js

```diff
@@ -52,7 +52,23 @@ const TableCell = props => {
         className="Tangelo__TableCell__Content"
         ref={content}
       >
-        {props.children}
+        {isEmpty(props.trailingIcons) || props.align !== 'right' || props.trailingIcons.map((icon, idx) => (
+          <div
+            className="Tangelo__TableCell__TrailingIconWrapper Tangelo__TableCell__TrailingIconWrapper--left"
+            key={`table_trailingicon_${rowIndex}_${columnIndex}_${idx}`}
+          >
+            {icon}
+          </div>
+        ))}
+        <span>{props.children}</span>
+        {isEmpty(props.trailingIcons) || props.align === 'right' || props.trailingIcons.map((icon, idx) => (
+          <div
+            className="Tangelo__TableCell__TrailingIconWrapper"
+            key={`table_trailingicon_${rowIndex}_${columnIndex}_${idx}`}
+          >
+            {icon}
+          </div>
+        ))}
```

Comment:
not sure if there's a better way to do this.

basically block of code for `trailingIcons` has to come before `{props.children}` when aligned right, and after `{props.children}` when aligned left/center so there's repetitive code.
Not sure if there's a way to refactor this though.

Let me know if you have better approach.